Update windows-store-for-business.md (#2461)
MicrosoftDocs/memdocs: memdocs/intune/apps/windows-store-for-business.md

- Offline apps that are free of charge can also be synced to Intune. These apps are installed by Intune, not by the store.
- To use this capability, devices must be joined to Active Directory Domain Services, Azure AD joined, or workplace-joined.
- Enrolled devices must be using the 1511 release of Windows 10 or later.
> [!NOTE]

> > Online MSFB apps can only be used for user context install, i.e., when deployed through Intune, you need to target user groups. Device licensed offline MSFB apps can be installed in device context, i.e., when deployed through Intune, you can target device groups as well as user groups.

Reviewer: ```suggestion
> Online Microsoft Store for Business apps can be used only for user context install; that is, when deployed through Intune, you need to target user groups. Device licensed offline Microsoft Store for Business apps can be installed in device context; that is, when deployed through Intune, you can target device groups as well as user groups.
```

Reviewer: minor edits
